app/views/thredded/messageboards/index.html.erb

```diff
@@ -3,9 +3,22 @@
 <% content_for :thredded_breadcrumbs, render('thredded/shared/breadcrumbs') %>
 <%= thredded_page do %>
   <section class="thredded--main-section thredded--messageboards">
-    <%= render @messageboards %>
+    <h3> Top Level Board </h3>
```

Comment:
Probably best to not render a title for top level boards because the categories are optional and it will look weird if there are no other categories.